src/api/core/operators/test_detect_text_in_image_tesseract.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import detect_text_in_image_tesseract
+
+class Test(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # initialize operator
+        detect_text_in_image_tesseract.initialize(param={})
+    
+    @classmethod
+    def tearDownClass(cls):
+        # delete config files
+        pass
+
+    def test_sample_image_from_disk(self):
+        image_path = "sample_data/text.png"
+        image_text = detect_text_in_image_tesseract.run(image_path)
+        expected_text = "It was the best of\ntimes, it was the worst\nof times, it was the age\nof wisdom, it was the\nage of foolishness..."
```

Comment:
@aatmanvaidya lets use a sample image with multilingual or non english text.

Reply:
will make the change